src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1329,6 +1329,27 @@ def test_aks_stop_and_start(self, resource_group, resource_group_location):
         start_cmd = 'aks start --resource-group={resource_group} --name={name}'
         self.cmd(start_cmd)
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_stop_and_start_private_cluster(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83316792&view=results) to validate the change.

Reply:
New [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83357757&view=results) for the affected case `test_aks_stop_and_start`